### ISSUE TITLE
Some questions about finetuning.

### ISSUE THREAD

**Issue by kawaiitakagi** [WeChatCV/WeDetect#15]
1 I observe that the grad_norms are always between 300-500 which seems too much large. 
2 I find that the losses (including total loss and other partial loss) do not decrease much. Total loss fluctuates at about 12.5.
Are these two phenomena normal? I am using the config and training scripts for COCO finetuning from this repository. Thanks a lot.

**Comment by fushh** [comment]
It is quite normal for the YOLO series. Below is our log.

<img width="1097" height="609" alt="Image" src="https://github.com/user-attachments/assets/cd5ad27f-783b-42b4-a7a1-3679b5374970" />
